src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if ((password == null || repeatPassword == null)
+                || !(password.equals(repeatPassword) && password.length() >= 10)) {
```

Comment:
do we need `repeatPassword == null` condition here? it is already covered by `password.equals(repeatPassword)`

Reply:
Null check condition moved up and throw PasswordValidationException.